main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Hand
+//  action 1: the hand grabs
+hand.grab();
+//  action 2: the hand curls into a fist
+hand.curlIntoFist();
+//  action 3: the hand claps
+hand.clap();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Tree
+//  action 1: the tree grows
+tree.grow();
+//  action 2: the tree sways
+tree.sway();
+//  action 3: the tree blossoms
+tree.blossom();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Window
+//  action 1: the window shows a reflection
+window.showReflection();
+//  action 2: the window shatters
+window.shatter();
+//  action 3: the window gets foggy
+window.getFoggy();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Umbrella
+//  action 1: the umbrella shields from the rain
+umbrella.shieldFromRain();
+//  action 2: the umbrella flutters in the wind
+umbrella.flutterInWind();
+//  action 3: the umbrella breaks due to heavy wind
+umbrella.breakFromWind();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Flower
+//  action 1: the flower blooms
+flower.bloom();
+//  action 2: the flower wilts
+flower.wilt();
+//  action 3: the flower bends towards sunlight
+flower.bendTowardSunlight();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Bicycle
+//  action 1: the bike wobbles
+bike.wobble();
+//  action 2: the bike bounces over terrain
+bike.bounceOverTerrain();
+//  action 3: the bike rusts
+bike.rust();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.